microsoft-365/solutions/b2b-extranet.md

```diff
@@ -27,6 +27,12 @@ This article walks through the steps to create a package of resources (in this c
 
 Before you begin, create the site or team that you want to share with the partner organization and enable it for guest sharing. See [Collaborate with guests in a site](collaborate-in-site.md) or [Collaborate with guests in a team](collaborate-as-team.md) for more information. We also recommend that you review [Create a secure guest sharing environment](create-secure-guest-sharing-environment.md) for information about security and compliance features that you can use to help maintain your governance policies when collaborating with guests.
 
+## License requirements
+
+[!INCLUDE [Azure AD Premium P2 license](../../../includes/active-directory-p2-license.md)]
+
+Specialized clouds, such as Azure Germany, and Azure China 21Vianet, are not currently available for use.
```

Comment:
```suggestion
Specialized clouds, such as Azure Germany and Azure China, are not currently available for use.
```

Reply:
My understanding is that "Azure China" is in fact "Azure operated by 21Vianet". If I'm wrong please ignore the edit.

Reply:
Thank you @JohanFreelancer9. I'll ignore the edit as the doc needs to have Azure China 21Vianet as per the standard.